File: CursoCsharpUdemy/Old/exer/EstruturaSequencial/Exer03.cs
using System;

namespace exer.udemy.EstruturaSequencial
{
    public class Exer03
    {
        static void Main(string[] args)
        {
            Console.Write("Informe o primeiro valor inteiro: ");
            int numberA = int.Parse(Console.ReadLine());

            Console.Write("Informe o segundo valor inteiro: ");
            int numberB = int.Parse(Console.ReadLine());

            Console.Write("Informe o terceiro valor inteiro: ");
            int numberC = int.Parse(Console.ReadLine());

            Console.Write("Informe o quarto valor inteiro: ");
            int numberD = int.Parse(Console.ReadLine());

            int difference = numberA * numberB - numberC * numberD;

            Console.Write($"Diferença: {difference}");
        }
    }
}

File: Curso de Java/Repeticao-Sequencial-Condicional/src/Exer/EstruturaSequencial/Exer03.java
package Exer.EstruturaSequencial;

import java.util.Scanner;

public class Exer03 {

	public static void main(String[] args) {
	
		Scanner sc = new Scanner(System.in);
		
		System.out.print("Primeiro valor: ");
		int a = sc.nextInt();
		System.out.print("Segundo valor: ");
		int b = sc.nextInt();
		System.out.print("Terceiro valor: ");
		int c = sc.nextInt();
		System.out.print("Quarto valor: ");
		int d = sc.nextInt();
		
		int diferenca = a * b - c * d;
		
		System.out.println("A diferença entre é: " + diferenca);
		
		sc.close();
	}

}
